### ISSUE TITLE
[Bug]: Game Stages-related FTB Chunks crash

### ISSUE THREAD

**Issue by ukmcplyr** [FTBTeam/FTB-Mods-Issues#880]
### Mod

FTB Chunks

### Mod version

ftb-chunks-forge-2001.1.3.jar

### Forge / Fabric version

1.20.1 47.1.28

### What issue are you having?

Hopefully got the right culprit, feel free to shout at me and I'll darken somebody else's door with this :grin:

Hit the following crash on a server for an in-dev modpack when teleporting from the overworld to the JAMD mining dimension with its portal:

```
[19Jul2023 17:27:05.497] [Server thread/ERROR] [net.minecraftforge.eventbus.EventBus/EVENTBUS]: Exception caught during firing event: 'void dev.ftb.mods.ftbchunks.integration.stages.forge.GameStagesStageHelperIm
pl.syncStages(net.minecraft.server.level.ServerPlayer)'
```

Reproduced in single player with FTB Chunks, JAMD, Game Stages plus deps only.

### Crashlogs

https://pste.ch/isagiludel.php

### Steps to reproduce

In single player with only:

- architectury-9.1.10-forge.jar
- Bookshelf-Forge-1.20.1-20.0.1.jar
- ftb-chunks-forge-2001.1.3.jar
- ftb-library-forge-2001.1.2.jar
- ftb-teams-forge-2001.1.4.jar
- GameStages-Forge-1.20.1-15.0.1.jar
- JAMD-Forge-3.0.0+mc1.20.1.jar
- Trenzalore-Forge-3.0.1+mc1.20.1.jar

1. Place JAMD portal block
2. Right click portal block
3. Crash

**Comment by desht** [comment]
Nope, you're right, that's a Chunks bug.  I'll get a fix out today.

**Comment by desht** [comment]
Fixed in 2001.1.4

https://www.curseforge.com/minecraft/mc-mods/ftb-chunks-forge/files/4654814

**Comment by ukmcplyr** [comment]
Thanks for such a speedy fix! :beer: :grin:
